src/main/java/io/grpc/annotations/checkers/ExperimentalApiChecker.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.annotations.checkers;
+
+import com.google.auto.service.AutoService;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.BugPattern.LinkType;
+import com.google.errorprone.BugPattern.SeverityLevel;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.matchers.Description;
+import com.sun.source.tree.Tree;
+import java.util.Map.Entry;
+import java.util.Optional;
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.element.AnnotationValue;
+import javax.lang.model.element.ExecutableElement;
+
+@AutoService(BugChecker.class)
+@BugPattern(
+    name = "ExperimentalApi",
```

Comment:
Do we need to worry about name conflicts with these plugins? I'm curious what happens if library X also releases an annotation checker named "ExperimentalApi" - would there be any way for a user to include both gRPC's "ExperimentalApi" checker and library X's "ExperimentalApi" checker? If not and the names would conflict, should the name here be changed something like "GrpcExperimentalApi"?

I know Guava's Beta API checker is just named "BetaApi", so maybe there's no need for including gRPC in the name here, but just wanted to check. 

@ejona86 any thoughts?

Reply:
I expect this name is used for squelching warnings with `@SuppressWarnings` as well as configuring how it is enabled (off vs error vs warning). We should probably include "grpc".

Reply:
I agree with ericgribkoff and ejona86. I think providing the explicit name would be helpful for users as well.

I will change checker names to `GrpcInternal` and `GrpcExperimentalApi`.